tests/test-forms-api.php

```diff
@@ -235,5 +235,35 @@ public function testManualFormulasTrimDuringSave ()
 		$this->assertSame( '2 + 2', $form['fields']['fld_60011111']['config']['manual_formula'] );
 	}
 
+	/**
+	 * Test manual Formulas trimming during forms saving
+	 *
+	 * @since 1.8.6
+	 *
+	 * @covers Caldera_Forms_Forms::save_form()
+	 */
+	public function testManualFormulasTrimDuringSaveWithRound ()
+	{
+		$form_id = $this->import_contact_form();
+		$form = Caldera_Forms_Forms::get_form($form_id);
+
+		$form['fields']['fld_60011111'] = [
```

Comment:
I know GitHub can make alignment all funky, but I want to make sure things are aligned properly even if GitHub doesn't show it properly. :)